### PR TITLE
Updated workflow Representation

### PR DESCRIPTION
**Pull request by arkajyotiMukherjee**
## Proposed changes
- Updated workflow Representation
- Added feature to change the layout
- Made it according to new design

Screenshot from 2020-12-23 17-47-02
Screenshot from 2020-12-23 17-47-14

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/ReduxRoot.tsx [hunk 1]
```diff
@@ -1,59 +1,54 @@
...
+//   ({ query }) => {
+//     const definition = getMainDefinition(query);
+//     return (
+//       definition.kind === 'OperationDefinition' &&
+//       definition.operation === 'subscription'
+//     );
+//   },
+//   authLink.concat(wsLink),
+//   authLink.concat(httpLink)
+// );
```
Comment (vanshBhatia-A4k9) [inline litmus-portal/frontend/src/ReduxRoot.tsx:42]: Please remove or use the commented code

File: litmus-portal/frontend/src/ReduxRoot.tsx [hunk 2]
```diff
@@ -1,59 +1,54 @@
...
 
-export const client = new ApolloClient({
-  link,
-  cache: new InMemoryCache(),
-});
+// export const client = new ApolloClient({
+//   link,
+//   cache: new InMemoryCache(),
+// });
```
Comment (vanshBhatia-A4k9) [inline litmus-portal/frontend/src/ReduxRoot.tsx:47]: same

File: litmus-portal/frontend/src/views/WorkflowDetails/ArgoWorkflow/index.tsx [hunk 3]
```diff
@@ -76,25 +94,55 @@ const ArgoWorkflow: React.FC<ArgoWorkflowProps> = ({ nodes }) => {
...
+          >
+            <AddRoundedIcon />
+          </ButtonOutline>
+          <ButtonOutline
+            className={classes.layoutButton}
+            handleClick={() => scrollGraph(false)}
+          >
+            <RemoveRoundedIcon />
+          </ButtonOutline>
+        </div> */}
```
Comment (vanshBhatia-A4k9) [inline litmus-portal/frontend/src/views/WorkflowDetails/ArgoWorkflow/index.tsx:121]: Please see if required.

**Review by vanshBhatia-A4k9 — COMMENTED** [review]
PTAL and change as appropriate.

**Review by vanshBhatia-A4k9 — APPROVED** [review]

**Review by amityt — APPROVED** [review]

**Review by S-ayanide — APPROVED** [review]

**Pull request merged**